airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -754,8 +754,6 @@ public void testCheckpointing() throws Exception {
 
   @Test
   @Order(14)
-  @DisabledIfEnvironmentVariable(named = "KUBE",
-                                 matches = "true")
   public void testRedactionOfSensitiveRequestBodies() throws Exception {
```

Comment:
turned this on for Kube since there Kube has a slightly different logging mechanism